imagepipeline/src/main/java/com/facebook/imagepipeline/producers/DecodeProducer.java

```diff
@@ -439,6 +443,10 @@ protected synchronized boolean updateDecodeJob(EncodedImage encodedImage, boolea
       return ret;
     }
 
+    private boolean isNotPartialData(EncodedImage encodedImage) {
```

Comment:
I don't really like this method name. Instead of having an `isNot` name I'd rather have e.g. `isPartialData()` and combine with `!` at the usage site.